lib/slack_transformer/html/newline.rb

```diff
@@ -0,0 +1,28 @@
+require 'nokogiri'
+
+module SlackTransformer
+  class Html
+    class Newline
+      attr_reader :input
+
+      def initialize(input)
+        @input = input
+      end
+
+      def to_slack        
+        fragment = Nokogiri::HTML.fragment(input)
+
+        fragment.children.each do |child|
+          case child.name
+          when 'p'
+            child.replace("\n #{child.children.to_html} \n")
```

Comment:
Do we want it wrapped in newlines, or just a newline at the end before the next tag to come along?

Reply:
<img width="842" alt="image" src="https://github.com/affinity/slack_transformer/assets/114527410/af83cfff-fe6b-4085-8594-fdf46a374353">

Reply:
hm i'm not sure i'm fully following your screenshots. i don't think that we should be adding in a newline before the <p> tag. And we may not need a newline at the end if there are no more elements after it 
so: `<p>Hi</p>` should just become `Hi`, but `Hello<p>World</p>` should become
```
Hello
World
```
And `Hello<p>World</p>Again` should become
```
Hello
World
Again
```